cake-autorate.sh

```diff
@@ -328,11 +331,13 @@ maintain_log_file()
 
 		t_log_file_start_us=${EPOCHREALTIME/.}
 
-		while read -r -N "${log_file_buffer_size_B}" -u "${log_fd}" log_chunk
```

Comment:
What was wrong with just adding `-t0` to peek? Also, I don't like how this code is structured:

```
while true
do
   while true
   do
````